src/Raven.Server/Documents/Handlers/Processors/Queries/AbstractQueriesHandlerProcessorForGet.cs

```diff
@@ -72,7 +72,7 @@ private async ValueTask IndexEntriesAsync(TQueryContext queryContext, TOperation
 
         HttpContext.Response.Headers[Constants.Headers.Etag] = CharExtensions.ToInvariantString(result.ResultEtag);
 
-        await using (var writer = new AsyncBlittableJsonTextWriter(context, RequestHandler.ResponseBodyStream(), token.Token))
+        await using (var writer = new AsyncBlittableJsonTextWriter(context, RequestHandler.ResponseBodyStream()))
```

Comment:
what is the implication here? during dispose, when we flush, we will not be able to cancel?